_i18n/template/resources/developer-guides/wallet-rpc.md

```diff
@@ -507,6 +611,9 @@ Return a list of incoming transfers to the wallet.
 Inputs:
 
 * *transfer_type* - string; "all": all the transfers, "available": only transfers which are not yet spent, OR "unavailable": only transfers which are already spent.
+* *account_index* - unsigned int; Return transfers for this account.
+* *subaddr_indices* - array of unsigned int; Return transfers sent to these subaddresses.
+* *verbose* - boolean; Enable verbose output, return key image if true.
```

Comment:
What a weird parameter.Documentation is correct, but I'll probably change it to make more sense.

Reply:
Ok, will wait for your update to make changes here, if you like.